src/components/decimal/decimal.spec.js

```diff
@@ -35,6 +36,14 @@ function render(props = {}, renderer = mount) {
   renderer(<Decimal {...defaultProps} />);
 }
 
+function renderI18n(locale) {
```

Comment:
This could use existing `mount` function, then tests could use existing `value()` method.

Reply:
good point, changed, thanks